src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -85,29 +72,18 @@ public void RegisterKnownConfigFiles(IDictionary<Language, IBindingConfig> langu
             }
         }
 
-        public IBindingConfig GetBindingConfig(Language language)
-        {
-            if (!bindingConfigInformationMap.TryGetValue(language, out var info) || info == null)
-            {
-                Debug.Fail("Expected to be called by the ProjectBinder after the known rulesets were registered");
-                return null;
-            }
-            return info;
-        }
-
         #endregion
 
         #region Public API
 
         public void Initialize()
         {
-            this.SolutionFullPath = this.projectSystem.GetCurrentActiveSolution().FullName;
+            // TODO CM cleanup
+            // no-op
```

Comment:
This method will be removed separately.

In fact, the `ISolutionBindingOperation` interface has three methods, only one of which is required. I'll continue to clean it up.